src/EventListener/ExceptionListener.php

```diff
@@ -72,7 +72,7 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         } else {
             switch (substr($code, 0, 1)) {
                 case '4':
-                    $message = 'Input Error';
+                    $message = $exception->getMessage();                    
```

Comment:
You're probably looking for this: https://github.com/kleijnweb/swagger-bundle/issues/27. If you check the exception type (should be InvalidParametersException), I guess `getMessage()` would be acceptable, even in `prod`.